pkg/combustion/templates/11-time-setup.sh.tpl

```diff
@@ -16,3 +16,31 @@ echo "pool {{ . }} iburst" >> /etc/chrony.d/eib-sources.conf
 {{ range .Servers -}}
 echo "server {{ . }} iburst" >> /etc/chrony.d/eib-sources.conf
 {{ end -}}
+
+{{ if .ForceWait -}}
+cat <<EOF >/etc/systemd/system/firstboot-timesync.service
```

Comment:
Can we include a comment describing the behaviour of `chrony-wait`? New developers wouldn't know why `firstboot-timesync.service` only contains an "echo" call.

Reply:
Where do you want this comment? The systemd unit is a workaround really, it enables us to force systemd to wait until its dependencies are satisfied before it lets k3s/rke2 start, and allows us to do it without modifying the standard k3s/rke2 systemd unit files.

Reply:
Yes, I fully understand this (with the help of the PR description).

Ideally, a comment just above this `cat` command saying something along the lines of "A oneshot service used to synchronise NTP by leveraging the default timeout (180 seconds) of chrony-wait" would be more than enough to convey the idea in the code itself.